modules/minimint-wallet/src/lib.rs

```diff
@@ -1206,7 +1225,7 @@ pub enum WalletError {
     #[error("Unknown block hash in peg-in proof: {0}")]
     UnknownPegInProofBlock(BlockHash),
     #[error("Invalid peg-in proof: {0}")]
-    PegInProofError(#[from] PegInProofError),
+    PegInProofError(PegInProofError),
```

Comment:
Any particular reason for reversing this recent change or just a rebase artifact?

Reply:
I must have missed this...for some reason `cargo +nightly fix` wanted to remove these.  I'll add it back.